### DISCUSSION TITLE
Database test fails with option --parallel

### DISCUSSION THREAD

**Discussion by markuskooche** [pestphp/pest#708] [Q&A]
I have the problem that these two tests only work if I run the tests without `--parallel`.
Is there any way to fix this or just run these two tests sequentially?

```php
it('asserts that it logs a message if transaction needs more than 2 seconds', function (): void {
    DB::transaction(function (): void {
        DB::statement('SELECT pg_sleep(1.9)');
        DB::statement('SELECT pg_sleep(0.2)');
    });

    // Assertion ...
})->group('provider', 'database', 'sequential');

it('asserts that it logs a message if query needs more than 1 second', function (): void {
    DB::statement('SELECT pg_sleep(1.1)');

    // Assertion ...
})->group('provider', 'database', 'sequential');
```

**Comment by flap152** [comment]
I commented the closed issue #819 referencing this discussion with a proposed approach for a PR that would implement sequential tests in otherwise-parallel testing. Hoping to get some ~~encouragement~~ feedback to proceed😅
(Or is the proper action to open a PR with not commit yet?)
